components/org.wso2.carbon.identity.oauth.par/src/main/java/org/wso2/carbon/identity/oauth/par/core/ParRequestBuilder.java

```diff
@@ -83,6 +83,9 @@ public boolean canHandle(HttpServletRequest request) {
                     .logDetailLevel(DiagnosticLog.LogDetailLevel.INTERNAL_SYSTEM);
             LoggerUtils.triggerDiagnosticLogEvent(diagnosticLogBuilder);
         }
+        if (canHandle) {
+            request.setAttribute(OAuthConstants.IS_PUSH_AUTHORIZATION_REQUEST, true);
```

Comment:
Lets add this to the buildRequest method so we know the request was build from it

Reply:
changed